modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/viewmodel/OnboardingLoginOrSignUpViewModel.kt

```diff
@@ -0,0 +1,18 @@
+package au.com.shiftyjelly.pocketcasts.account.viewmodel
+
+import android.content.Context
+import androidx.lifecycle.ViewModel
+import au.com.shiftyjelly.pocketcasts.account.BuildConfig
+import au.com.shiftyjelly.pocketcasts.utils.extensions.isGooglePlayServicesAvailableSuccess
+import com.google.android.gms.common.GoogleApiAvailability
+import dagger.hilt.android.lifecycle.HiltViewModel
+import dagger.hilt.android.qualifiers.ApplicationContext
+import javax.inject.Inject
+
+@HiltViewModel
+class OnboardingLoginOrSignUpViewModel @Inject constructor(
+    @ApplicationContext private val context: Context
```

Comment:
Nitpick, but we don't actually need to hold onto the context (at least we don't need to yet), so we could drop the `private val`. If we want to keep ahold of it, perhaps it would be worth adding `@SuppressLint("StaticFieldLeak")` to get rid of the lint warning since it's not possible for a ViewModel to "leak" the application's context.

Reply:
Oops. Thanks, I have removed `private val`.